--- index.orig.js	2018-01-25 19:24:58.000000000 +0200
+++ index.js	2018-03-30 12:26:08.000000000 +0300
@@ -6,7 +6,7 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * 
+ *
  * @format
  */
 
@@ -132,8 +132,10 @@
     // We whitelist only what would work. For example `--inspect` doesn't work
     // in the workers because it tries to open the same debugging port. Feel
     // free to add more cases to the RegExp. A whitelist is preferred, to
+    var a = process.execArgv;
+    a.push("--max-old-space-size=8192");
     // guarantee robustness when upgrading node, etc.
-    const execArgv = process.execArgv.filter(
+    const execArgv = a.filter(
     arg =>
     /^--stack[_-]trace[_-]limit=[0-9]+$/.test(arg) ||
     /^--heap[_-]growing[_-]percent=[0-9]+$/.test(arg) ||
@@ -193,4 +195,4 @@
   constructor(message) {
     super(message);this.type = 'TransformError';
     Error.captureStackTrace && Error.captureStackTrace(this, TransformError);
-  }}
\ No newline at end of file
+  }}
